From acb2f01cabd310c193942db7e87635f153d9e9ea Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lawrence=20Vel=C3=A1zquez?= <larryv@macports.org>
Date: Mon, 9 Feb 2015 22:50:13 -0500
Subject: [PATCH 05/16] (MP) BuildSystem: Don't abort on missing GCC
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Trace mode hides all `gcc` executables from the configure script if
`configure.compiler` is “`clang`”, causing it to fail. Make this
non-fatal because we pass the correct compiler using `--gcc` anyway.
---
 make/configure.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/make/configure.py b/make/configure.py
index 1bac087..b968a18 100644
--- a/make/configure.py
+++ b/make/configure.py
@@ -1403,7 +1403,7 @@ try:
         ar    = ToolProbe( 'AR.exe',    'ar' )
         cp    = ToolProbe( 'CP.exe',    'cp' )
         curl  = ToolProbe( 'CURL.exe',  'curl', abort=False )
-        gcc   = ToolProbe( 'GCC.gcc',   'gcc', IfHost( 'gcc-4', '*-*-cygwin*' ))
+        gcc   = ToolProbe( 'GCC.gcc',   'gcc', IfHost( 'gcc-4', '*-*-cygwin*' ), abort=False)
 
         if host.match( '*-*-darwin*' ):
             gmake = ToolProbe( 'GMAKE.exe', 'make', 'gmake' )
-- 
2.3.0

